plugins/vega/web_client/views/VegaWidget.js

```diff
@@ -29,12 +30,11 @@ var VegaWidget = View.extend({
                 url: `item/${this.item.id}/download`
             })
                 .done(function (spec) {
-                    vg.parse.spec(spec, function (chart) {
-                        chart({
-                            el: '.g-item-vega-vis',
-                            renderer: 'svg'
-                        }).update();
-                    });
+                    let runtime = parse(spec);
+                    let view = new VegaView(runtime)
+                        .initialize(document.querySelector('.g-item-vega-vis'))
```

Comment:
@waxlamp @manthey Just pointing out that we should be using jQuery (ideally, Backbone-view-instance-scoped jQuery) to do DOM selection. Using `document.querySelector` is inconsistent with Girder / Backbone style. If you need a raw HTMLElement, you can always use `[0]` on the jQuery selection.

There are a number of places in this PR with this issue. Ideally, these should be cleaned up to discourage reuse.